pkg/ottl/contexts/ottlmetric/README.md

```diff
@@ -0,0 +1,34 @@
+
+## Paths
+In general, the Metric Context supports accessing pdata using the field names from the [metrics proto](https://github.com/open-telemetry/opentelemetry-proto/blob/main/opentelemetry/proto/metrics/v1/metrics.proto).  All integers are returned and set via `int64`.  All doubles are returned and set via `float64`.
+
+The following fields are the exception.
+
+| path                                   | field accessed                                                                 | type                                                                    |
+|----------------------------------------|--------------------------------------------------------------------------------|-------------------------------------------------------------------------|
+| resource                               | resource of the metric being processed                                         | pcommon.Resource                                                        |
+| resource.attributes                    | resource attributes of the metric being processed                              | pcommon.Map                                                             |
+| resource.attributes\[""\]              | the value of the resource attribute of the metric being processed              | string, bool, int64, float64, pcommon.Map, pcommon.Slice, []byte or nil |
+| instrumentation_scope                  | instrumentation scope of the metric being processed                            | pcommon.InstrumentationScope                                            |
+| instrumentation_scope.name             | name of the instrumentation scope of the metric being processed                | string                                                                  |
+| instrumentation_scope.version          | version of the instrumentation scope of the metric being processed             | string                                                                  |
+| instrumentation_scope.attributes       | instrumentation scope attributes of the metric being processed                 | pcommon.Map                                                             |
+| instrumentation_scope.attributes\[""\] | the value of the instrumentation scope attribute of the metric being processed | string, bool, int64, float64, pcommon.Map, pcommon.Slice, []byte or nil |
+
+## Enums
+
+The Metrics Context supports the enum names from the [metrics proto](https://github.com/open-telemetry/opentelemetry-proto/blob/main/opentelemetry/proto/metrics/v1/metrics.proto).
+
+In addition, it also supports an enum for metrics data type, with the numeric value being [defined by pdata](https://github.com/open-telemetry/opentelemetry-collector/blob/main/pdata/pmetric/metrics.go).
```

Comment:
We might need an example of how to use this?

Reply:
Do you mean how to use the metric data type enum?